### DISCUSSION TITLE
Force users to re-login if they are inactive in X minutes

### DISCUSSION THREAD

**Discussion by emilsjog** [abpframework/abp#9220] [Q&A]
Hi,

What is the recommended way to implement functionality for automatic logout after a configured time of inactivity?
I just tried to change `AbsoluteRefreshTokenLifetime` and other values in the DB but there was no effect in the blazor-UI. 

Abp version **4.3**
UI: **blazor-server**

**Comment by mz-s73** [comment]
@emilsjog Did you find answer to this ?
